### DISCUSSION TITLE
Import SVG and then edit individual elements

### DISCUSSION THREAD

**Discussion by mariusa** [excalidraw/excalidraw#6082] [Q&A]
Hi & thanks for this wonderful library!

I tried importing a SVG, which works fine. Similar to a raster image, it can only be moved as a block.

How can I ungroup and edit individual SVG elements? Similar to how Libraries > Stick people works (eg I can modify each hand).
Would one need to use this https://github.com/excalidraw/svg-to-excalidraw ? And if so, why isn't it enabled by default for the online editor?

**Comment by zsviczian** [comment]
you can try Excalidraw-Obsidian. The SVG import is implemented there.
You can simply ungroup elements after import and edit parts of the SVG import. 
See it in action here: https://youtu.be/vlC1-iBvIfo

**Reply by mariusa** [reply]
It looks great there! Glad you procrastinated :))  
What would be the steps to provide the same functionality in excalidraw native? (web editor)

**Comment by scmdavid** [comment]
I tried SVG from [Azure architecture icons](https://learn.microsoft.com/en-us/azure/architecture/icons/), but it did not work
